docs/src/topics/SimpleCellValues_literate.jl

```diff
@@ -0,0 +1,95 @@
+# We start by including `Ferrite` and `Test`, 
+# to allow us to verify our implementation.
+using Ferrite, Test
+
+# Define a simple version of the cell values object, which only supports
+# identity mapping of scalar interpolations, without embedding. 
```

Comment:
```suggestion
# Define a simple version of the cell values object, which only supports
# scalar interpolations using an identity mapping between physical and
# reference elements. Here we assume that the element has the same
# dimension as the physical space, which excludes for example surface
# elements.
```
Maybe we should expand here a bit, because this is user-facing API and we cannot assume the same level of knowledge. Maybe we can add some link for the identity mapping and the surface element with further explanation for interested users.